src/ingest_validation_tools/table-schemas/level-2/lcms.yaml

```diff
@@ -27,6 +27,11 @@ fields:
   -
     name: mass_range_high_value
     description: The high value of the scanned mass range for MS1. (unitless)
+  -
+    name: mass_range_unit
+    description: Units of the mass range values
+      enun
+       - m/z
```

Comment:
- mass range is literally unitless.

Reply:
Yes. It is a ratio of the mass value divided by the charge value. m/z describes how the mass is calculated. mass spectrometry output is called "m/z", not just "mass". z is usually 1 but not always.